package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@coingecko/cryptoformat": "^0.4.4",
     "@loadable/component": "^5.15.0",
+    "@metamask/onboarding": "^1.0.1",
```

Comment:
question(non-blocking): from the MetaMask [onboarding-diagram](https://docs.metamask.io/guide/onboarding-library.html#onboarding-diagram) it seems like we would also need the `@metamask/forwarder` package to take full advantage of this package?
I did a quick test with Chrome and can confirm that the redirect to the portal does not work after successfull initial setup of metamask.
We could maybe track this in an issue, I think it would greatly improve usability - but is probably out of scope of this fix.

Reply:
thought(blocking): one additional remark. For this PR it seems to be enough to implement the function isMetaMaskInstalled() directly into our code base to remove the dependency to a new package. As mentioned aboe, I think it can improve our usability, but would need some additional attention for that to work.

From the MetaMask package:
```ts
  /**
  * Checks whether the MetaMask extension is installed
  */
  static isMetaMaskInstalled() {
    return Boolean(
      (window as any).ethereum && (window as any).ethereum.isMetaMask,
    );
  }
```

Reply:
I just tried to do it without the library, the problem is that after the installation (without a refresh) the page is not aware that a new extension has been added, so the `isMetaMaskInstalled()` function returns false even if the package is installed.

Reply:
The redirect is very unreliable for some reason, but from what I read in the @metamask/forwarder [readme](https://github.com/MetaMask/forwarder) file it seems like it's already included in the onboarding library, you don't have to install it on its own.